README.md

```diff
@@ -79,22 +85,9 @@ $ kubectl create -f ./example/pvc-tier2.yaml
 
 #### Install a Pravega cluster
 
-Use Helm to install a sample Pravega cluster with release name `bar`.
-
-```
-$ helm install bar charts/pravega --set zookeeperUri=[ZOOKEEPER_HOST] --set bookkeeperUri=[BOOKKEEPER_SVC] --set storage.longtermStorage.filesystem.pvc=[TIER2_NAME]
-```
-
-where:
-
-- **[ZOOKEEPER_HOST]** is the host or IP address of your Zookeeper deployment (e.g. `zookeeper-client:2181`). Multiple Zookeeper URIs can be specified, use a comma-separated list and DO NOT leave any spaces in between (e.g. `zookeeper-0:2181,zookeeper-1:2181,zookeeper-2:2181`).
-- **[BOOKKEEPER_SVC]** is the is the name of the headless service of your Bookkeeper deployment (e.g. `bookkeeper-bookie-0.bookkeeper-bookie-headless.default.svc.cluster.local:3181,bookkeeper-bookie-1.bookkeeper-bookie-headless.default.svc.cluster.local:3181,bookkeeper-bookie-2.bookkeeper-bookie-headless.default.svc.cluster.local:3181`).
-- **[TIER2_NAME]** is the longtermStorage `PersistentVolumeClaim` name. `pravega-tier2` if you created the PVC above.
-
-
-Check out the [Pravega Helm Chart](charts/pravega) for the complete list of configurable parameters.
+To understand how to deploy a pravega cluster using helm, refer to [this](charts/pravega#installing-the-chart).
 
-Verify that the cluster instances and its components are being created.
+Once the pravega cluster with release name `bar` has been created, use the following command to verify that the cluster instances and its components are being created.
```

Comment:
Can we change the default name of pravegacluster release name to "pravega" as that's what is given as default pravega name in configmap for bookkeeper by default.

Reply:
By giving a different release name, we are able to show that release name and cluster name might not always be the same

Reply:
I know that but in the bookkeeper configmap the name for pravega cluster is **pravega** so for a user deploying for the first time this will create an issue if he deploys pravega with any other release name other than pravega.